js/main.js

```diff
@@ -1,28 +1,68 @@
-/*
-Apple
-The apple tree (Malus pumila, commonly and erroneously called Malus domestica) is a deciduous tree in the rose family best known for its sweet, pomaceous fruit, the apple. It is cultivated worldwide as a fruit tree, and is the most widely grown species in the genus Malus. The tree originated in Central Asia, where its wild ancestor, Malus sieversii, is still found today. Apples have been grown for thousands of years in Asia and Europe, and were brought to North America by European colonists. Apples have religious and mythological significance in many cultures, including Norse, Greek and European Christian traditions.
+/* global $ */
 
-Banana
-The banana is an edible fruit – botanically a berry – produced by several kinds of large herbaceous flowering plants in the genus Musa. In some countries, bananas used for cooking may be called plantains, in contrast to dessert bananas. The fruit is variable in size, color and firmness, but is usually elongated and curved, with soft flesh rich in starch covered with a rind which may be green, yellow, red, purple, or brown when ripe. The fruits grow in clusters hanging from the top of the plant. Almost all modern edible parthenocarpic (seedless) bananas come from two wild species – Musa acuminata and Musa balbisiana. The scientific names of most cultivated bananas are Musa acuminata, Musa balbisiana, and Musa × paradisiaca for the hybrid Musa acuminata × M. balbisiana, depending on their genomic constitution. The old scientific name Musa sapientum is no longer used.
+console.log('Hello World');
+		
+function showActive(event) {
+  event.preventDefault();
 
-Cherry
-The cherry fruits of commerce usually are obtained from a limited number of species such as cultivars of the sweet cherry, Prunus avium. The name 'cherry' also refers to the cherry tree, and is sometimes applied to almonds and visually similar flowering trees in the genus Prunus, as in "ornamental cherry", "cherry blossom", etc. Wild cherry may refer to any of the cherry species growing outside of cultivation, although Prunus avium is often referred to specifically by the name "wild cherry" in the British Isles.
+  $('li').removeClass('active');
+  $(this).addClass('active');
```

Comment:
Interesting approach using many functions to solve the problem.

An alternative solution would be to use a single function like so:

```js
function handleSelect() {
  $('li').removeClass('active');
  $(this).addClass('active');
  var fruit = $(this).text(); // extracts the text of the selected item e.g. 'Grape' or 'Cherry'
  var description = '';
  if (fruit === 'Grape') {
    description = 'The grape is a fruit blah blah blah';
  } else if (fruit === 'Cherry') {
    description = 'The cherry is a fruit blah blah blah';
  } else if (/* etc ... */) {
  }
  $('#title').html(fruit); // sets the text we pulled out from the beginning to the title
  $('#description').html(description); // sets the description to the dynamically assigned description variable above
}

$('li').click(handleSelect);
```